graphics/fontforge/Portfile

```diff
@@ -20,68 +22,19 @@ long_description \
     some Type0), TrueType, OpenType (Type2), CID, SVG.  Previously known as \
     pfaedit.
 
-homepage        http://fontforge.org/
-master_sites    sourceforge:project/fontforge/fontforge-source \
-                sourceforge:project/fontforge/fontforge-docs:docs \
-                ${homepage}:cidmaps \
-                http://download.savannah.gnu.org/releases/freetype/:freetype \
-                sourceforge:project/freetype/freetype2/${ftversion}:freetype
-distname        ${name}_full-${version}
-dist_subdir     ${name}/${version}
-use_bzip2       yes
-
-set docdistfile ${name}_htdocs-${docversion}${extract.suffix}
+homepage        http://fontforge.github.io/
 
-distfiles-append \
-                cidmaps.tgz:cidmaps \
-                ${docdistfile}:docs
-checksums       ${distname}${extract.suffix} \
-                rmd160  351af6022d1c1c80e61788b699ff882bcde1b55f \
-                sha256  e5cb9e9e64a99d1dfc549950741eb9b44c3162a4c62f71053905c1f2d024295f \
-                cidmaps.tgz \
-                rmd160  d194355e61813d31fe4813bdf10a62d66a9f6d04 \
-                sha256  1bf9c7eb8835e6ed94e62cb49f1141bc046c562849e52e6c3c7f1d7cfc95c7b3 \
-                ${docdistfile} \
-                rmd160  8afd7d4eb9d590c989f16047e3ff4c926d1eab31 \
-                sha256  a938c08e66c1ff1e853e0833fbde6cb5d92d900f66b1958b6769f5b7d32ad1e7 \
-                freetype-${ftversion}${extract.suffix} \
-                rmd160  9da7fcde9c42f81c399201d055d85109b2ae7814 \
-                sha256  0c8e242c33c45928de560d7d595db06feb41d1b22167e37260ceabe72f9e992f
+checksums       rmd160  af473f403999937dd95715bdfd98f582d05c94cd \
+                sha256  840adefbedd1717e6b70b33ad1e7f2b116678fa6a3d52d45316793b9fd808822 \
```

Comment:
We are nowadays trying to add `size` to checksums.

Reply:
hmm, IMO this is an overkill. if the checksum is not enough then something is wrong. 

If you really want to improve the integrity, I suggest to sign the packages with a gpg key. (automated after they are merged to master)

Reply:
Setting the size is not a matter of integrity checking, but a matter of (future) ability to estimate download size and time.